index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+    />
+  </head>
+  <body>
+    <header id="header">
+      <div class="logo">
+        <a href="index.html"
+          ><h1><span>K</span>-Kal</h1></a
+        >
+      </div>
+      <nav class="nav">
+        <div class="nav-container">
+          <a href="#menu" class="nav-menu"
+            ><i class="fa-solid fa-bars nav-icon"></i
+          ></a>
+          <a href="#" class="nav-menu nav-menu-second"
+            ><i class="fa-solid fa-x nav-icon"></i
+          ></a>
+          <ul class="side-bar" id="menu">
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section class="section-p1">
+        <div class="div1">
+          <h2>I'm Cristhian. Glad to see you!</h2>
+          <p>
+            I'm a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don't hesitate to
+            contact me.
+          </p>
+        </div>
```

Comment:
- [ ] [Optional] I would suggest you please remove the border to stick to the Figma design.
- [ ] [Optional] I would suggest you please stick to the design as much as possible like spacing, color, etc. The logo and Humburg buttons should be aligned with the content. Please see the image below
- [ ] [Optional] I would suggest you please use this `color: #172B4D;` with `h2` tag to match the design.
### Your design
![hia](https://user-images.githubusercontent.com/81727988/234943454-62a3bd9a-0067-45a2-8830-806537212e0f.png)